### PR TITLE
Explicitly set command and args.

### PR DESCRIPTION
**Pull request by spxtr**
I'm getting a very confusing error for this job alone. This change is a stab in the dark, but might fix it.

### CODE REVIEW

File: prow/jobs.yaml [hunk 1]
```diff
@@ -200,6 +200,8 @@ kubernetes/test-infra:
     containers:
     - name: builder
       image: gcr.io/k8s-testimages/test-infra-go-test:0.0
+      command: ["/bin/bash", "-c"]
+      args: ["git clone https://github.com/kubernetes/test-infra && ./test-infra/jenkins/bootstrap.py --repo=k8s.io/kubernetes --pull=${PULL_REFS} --job=pull-test-infra-prow-test"]
```
Comment (fejta) [inline prow/jobs.yaml]: Why isn't this in the image?

**Comment by spxtr** [comment]
It's super ugly, and if this does fix it then we need to figure out why.

File: prow/jobs.yaml (hunk 1 above)
Comment (spxtr) [inline prow/jobs.yaml]: It is, but I'm getting a super low-level docker error and it's one of the few differences between this image and the bazel build image, which is working properly.

I'm overriding it here to cut down on the differences.

**Pull request merged**